### PR TITLE
Changed the fotos tab to video announcement.

### PR DESCRIPTION
**Pull request by cybit**
Did not change the picture though.
The playlist link does not work on purpose so we can prepare the general announcement without leaks.

### CODE REVIEW

File: src/content/deadlines/04_photos.mdx [hunk 1]
```diff
@@ -1,8 +1,10 @@
...
+title: 2024 Videos
+subtitle: Upgrade your experience
+url: https://www.youtube.com/playlist?list=
 image: "./enjoy.jpg"
 ---
 
-Remember the days at the conference? Have a look at our photo gallery to recall all those nice memories.
+Any sessions you wanted to attend but could not due to too many parallel tracks? Or maybe you missed a session because you were too tired or had to take care of something else?
+
+Go to our playlist and watch 3 days of EuroPython 2024 sessions. Including all keynotes, talks, and lightning talks.
```
Comment (hypha) [inline src/content/deadlines/04_photos.mdx:10]: To me, this is way too wordy for CTA. Maybe something like "Missed a session or couldn’t make it to the conference? Dive into the recordings and immerse yourself in 3 days of inspiring talks, keynotes, and lightning talks. Learn together and relish in our shared love for Python!" (You can even omit the first sentence)
I think it is generally good not to have multiple paragraphs for the CTA.

File: src/content/deadlines/04_photos.mdx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 ---
-title: Photos
-subtitle: Refresh you memories!
-url: /photos
+title: 2024 Videos
+subtitle: Upgrade your experience
```
Comment (hypha) [inline src/content/deadlines/04_photos.mdx:3]: Personally, this doesn't sound very good to me. It suggests my experience at the conference was not as good. I prefer "Relive EuroPython 2024"

File: src/content/deadlines/04_photos.mdx (hunk 1 above)
Comment (hypha) [inline src/content/deadlines/04_photos.mdx:10]: To me, this is way too wordy for CTA. Maybe something like "Missed a session or couldn’t make it to the conference? Dive into the recordings and immerse yourself in 3 days of inspiring talks, keynotes, and lightning talks. Learn together and relish in our shared love for Python!" (You can even omit the first sentence)
I think it is generally good not to have multiple paragraphs for the CTA.

File: src/content/deadlines/04_photos.mdx [hunk 3]
```diff
@@ -1,8 +1,10 @@
 ---
-title: Photos
-subtitle: Refresh you memories!
-url: /photos
+title: 2024 Videos
```
Comment (hypha) [inline src/content/deadlines/04_photos.mdx:2]: If you replace the photos with videos, I think there should still be a prominent place for photos on the homepage (footer is not prominent enough). 

Personally, I think it actually makes sense to replace photos with schedule. And then put schedule here on the secondary CTA here: https://github.com/EuroPython/website/blob/2c1254dac5ba79a46adf671f5c2372f6ccf37bd0/src/pages/index.astro#L56-L64 
Or you can put photos there, whichever you prefer.

File: src/content/deadlines/04_photos.mdx (hunk 2 above)
Comment (cybit) [reply src/content/deadlines/04_photos.mdx]: Changed it to your wording.

File: src/content/deadlines/04_photos.mdx (hunk 1 above)
Comment (cybit) [reply src/content/deadlines/04_photos.mdx]: Changed it to your sentences. I wrote new sentences to differentiate from last years page. But seems to have gone awry.

File: src/content/deadlines/04_photos.mdx (hunk 3 above)
Comment (cybit) [reply src/content/deadlines/04_photos.mdx]: I moved the Photos down there for two reasons:
1. I wanted to have the schedule next to the recordings so visitors don't have to play connect the dots.
2. I was too lazy to create a new card picture for the recordings.

File: src/pages/index.astro [hunk 4]
```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
...
-          On-site Volunteer
+          Stills and behind the scenes
         </Title>
 
         <ButtonLink
-          href="/volunteers"
+          href="/Photos"
         >
-          Join Us in Shaping EuroPython 2024
+          Peruse the conference and social event photos
```
Comment (hypha) [inline src/pages/index.astro:63]: I think the title should be more obvious and short, e.g. 2024 Photos. You want people from most backgrounds to immediately recognise what this is about and click on it :) The button text can be longer and descriptive like what you have or  e.g. Conference stills and behind-the-scenes moments

File: src/pages/index.astro [hunk 5]
```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
         />
 
         <Title level={3} className="font-bold text-3xl mb-4 mt-12 lg:mt-auto">
-          On-site Volunteer
+          Stills and behind the scenes
         </Title>
 
         <ButtonLink
-          href="/volunteers"
+          href="/Photos"
```
Comment (hypha) [inline src/pages/index.astro:61]:
```suggestion
          href="/photos"
```

File: src/pages/index.astro (hunk 4 above)
Comment (cybit) [reply src/pages/index.astro]: I did some adjustments. Better? 
I don't want to have too much text in the button and I want to have an action indicated in the text as well. So even now I think the text is too long.

Comment (hypha) [reply src/pages/index.astro]: Looks OK to me, but I agree that it shorter would be ideal. Ideas: "Moments of Togetherness" or "Captured Connections".

Comment (cybit) [reply src/pages/index.astro]: i did use a combination of our short DM chat for the button text. Thank you.

**Comment by hypha** [comment]
Not a request, but a question: would you want to list the video publishing date on the FAQ milestone?

**Review by egeakman — APPROVED** [review]
Thanks Cyril!
